### PR TITLE
update set-dfsreplicatedfolder.md

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #525

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Minor copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips

Peer review needed for this PR. Thanks.

File: docset/windows/dfsr/set-dfsreplicatedfolder.md [hunk 1]
```diff
@@ -131,8 +131,7 @@ Accept wildcard characters: False
 ### -DfsnPath
 Specifies the DFS Namespace folder path of the replicated folder.
 
-The DFS Namespace folder path has no effect on replication.
-This property exists only as a descriptive convenience to administrators and DFS Replication service does not validate the value of this property.
+This parameter is used to connect replication group to the replicated folder.
```
Comment (JohanFreelancer9) [inline docset/windows/dfsr/set-dfsreplicatedfolder.md]:
```suggestion
This parameter is used to connect the replication group to the replicated folder.
```

**Comment by get-itips** [comment]
Ok

**Comment by dariomws306** [comment]
I´m ok with this.

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9 , your recommendation was successfully applied, thank you!

**Comment by e0i** [comment]
@AndreaBarr This PR can be merged. Thanks.

**Pull request merged**